doc/manual/installation/uninstalling-nix.xml

```diff
@@ -0,0 +1,56 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      version="5.0"
+      xml:id="ch-uninstalling-nix">
+
+<title>Uninstalling Nix</title>
+
+<para>The following steps describes how to remove Nix from your system.</para>
+
+<para>First of all you need to remove the <filename>/nix</filename>
+directory:</para>
+
+<screen>sudo rm -rf /nix</screen>
+
+<para>You also need to remove some files from your home directory:</para>
+
+<screen>
+rm -rf $HOME/.nix-profile
+rm -rf $HOME/.nix-defexpr
+rm -rf $HOME/.nix-channels
+rm -rf $HOME/.config/nixpkgs
```

Comment:
These files do not need to be removed.

Reply:
But shouldn't they be? If I was testing out some new software on my machine, I wouldn't want it to leave behind unnecessary files or directories. Perhaps we could move these commands into a sub-section describing how to completely purge Nix from your system?

Reply:
It may be better to have a section which lists all files that are (typically) existing/generated by Nix and what their purpose is. Based on that a reader can easily decide what they need to remove.